### PR TITLE
Testimonials Screen Shot Update LinkFree to BioDrop

### PR DESCRIPTION
**Pull request by chiragthakur09**
## Fixes Issue
Closes https://github.com/EddieHubCommunity/BioDrop/issues/8814

## Changes proposed

- [ ] New Issue- of type Testimonial Screenshot

### CODE REVIEW

**Comment by sital002** [comment]
Issue link https://github.com/EddieHubCommunity/BioDrop/issues/8814

**Review by sital002 — COMMENTED** [review]
After changes: 
image

**Comment by ChinmayMhatre** [comment]
@chiragthakur09 any reason you have marked this pr as draft?

File: data/ChiragThakur.json
Comment (sital002) [inline data/ChiragThakur.json:1]: Please create different Pull Requests for creating a profile on biodrop

**Comment by SaraJaoude** [comment]
Due to lack of activity from the author I am closing this PR.

**Pull request closed without merging**